rtc-video-quality/visual_metrics.py

```diff
@@ -460,3 +460,10 @@ def HandleFiles(variables):
       formatters = "%s   formatter.format(better, %d);" % (formatters, i+1)
 
   return FillForm(page_template, vars())
+
+if __name__ == "__main__":
+    if len(sys.argv) < 3:
+      print(HandleFiles.__doc__)
+    else:
+      print(HandleFiles(sys.argv))
```

Comment:
Please move this into a function called main().

https://google.github.io/styleguide/pyguide.html#317-main

Reply:
Updated